src/components/earn/PoolCard.tsx

```diff
@@ -39,7 +42,7 @@ const StatContainer = styled.div`
 const Wrapper = styled(AutoColumn)<{ showBackground: boolean; bgColor: any }>`
   border-radius: 12px;
   width: 100%;
-  overflow: hidden;
+  overflow: unset;
```

Comment:
why was this added?

Reply:
![image](https://user-images.githubusercontent.com/95660534/163280839-91f7454b-d69d-4a1b-9145-cd9fb811e3c7.png)
To show the whole close button on the pool card, I've changed it and it doesn't affect other features or styles for the pool card.